cluster/ci/monitoring/mixins/grafana_dashboards/dptp.jsonnet

```diff
@@ -39,4 +39,21 @@ dashboard.new(
     x: 0,
     y: 0,
   })
+.addPanel(
+    (graphPanel.new(
+        'Plank Infra-Jobs Failures',
+        description='sum(rate(prowjob_state_transitions{job="plank",job_name!~"rehearse.*",state="failure"}[5m])) by (job_namespace, job_name) * on (job_namespace, job_name) group_right prow_job_labels{job_agent="kubernetes",label_ci_openshift_io_role="infra"}',
```

Comment:
I don't think we need to partition by namespace? We just want the number of jobs named `foo` that failed, right?

Reply:
I think using namespace is even a bug i made.
We did have some failures ... without namespace ... they do show up.

https://grafana-prow-monitoring-stage.svc.ci.openshift.org/d/8ce131e226b7fd2901c2fce45d4e21c1/dptp-dashboard?orgId=1&fullscreen&panelId=3

Why `namespace` hides my failures? I still cannot explain.
Started reading [doc](https://prometheus.io/docs/prometheus/latest/querying/operators/) ... the 10th time

Reply:
Haha ... I think i know why now ...
periodic jobs have no namespace as a label ... does it sound correct?
I feel odd though :) Will discuss with you tomorrow morning @stevekuznetsov

Reply:
Actually it is the problem of `prowjob_state_transitions`, not `prow_job_labels`.

This should fix it.